docs/source/restore/compatibility-matrix.rst

```diff
@@ -0,0 +1,16 @@
+Compatibility Matrix
+====================
+
+The following table shows which version of Scylla Manager restore task supports which versions of Scylla.
+
+
+.. list-table::
+   :widths: 25 25 25
+   :header-rows: 1
+
+   * - Scylla Manager Version
+     - Scylla Open Source Version
+     - Scylla Enterprise Version
+   * - 3.1
+     - 5.0, 5.1, 5.2
+     - 2020.1, 2021.1, 2022.1, 2022.2, 2023.1
```

Comment:
I added the compatibility matrix, but I am not sure how to test if restore works well with enterprise scylla.

Reply:
LGTM

Reply:
ScyllaDB Enterprise 2021.1 is based on 4.3, so how how come it is supportd?

Reply:
The earliest supported enterprise version is now 2022.1.